src/button/button-icon.scss

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+@mixin iui-button-icon {
+  &:not(.iui-user-icon) {
+    width: $iui-icons-default;
+    height: $iui-icons-default;
+    @media (prefers-reduced-motion: no-preference) {
```

Comment:
a fill transition does not really count as "motion" so I think this media query could be removed